src/poggit/module/build/SelfBuildPage.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
```

Comment:
If you look at a fully populated page it's so obvious that they are projects there's no need to repeat the word 20 times as, not as a heading at least, or at the start of the box